internal/settings/settings.go

```diff
@@ -144,6 +144,7 @@ func (s *Settings) SetAutoupdate(autoupdate string) {
 
 func (s *Settings) SetLastUpdateCheck(t int64) {
 	config := viper.GetStringMap("config")
+	fmt.Println(config)
```

Comment:
This shouldn't be here.

Reply:
It is just for debugging purposes :)

Reply:
I mean, I'm just testing here bc I can't reproduce in my local env

Reply:
I figured but we shouldn't merge it, should we?

Reply:
Nope!